src/ui/src/services/ipc-client.ts

```diff
@@ -169,6 +175,9 @@ declare global {
         get: () => Promise<GeneralSettings>;
         setMode: (mode: ToolApprovalMode) => Promise<{ success: boolean }>;
       };
+      portal: {
+        getMyShaves: () => Promise<{ success: boolean; data?: unknown; error?: string }>;
```

Comment:
why is the data unknown? you know what data it returns, you or someone created the api endpoint, just create the correct type based of the data the api returns